CMakeLists.txt

```diff
@@ -718,7 +721,7 @@ elseif(CMAKE_SYSTEM_NAME MATCHES "Windows")
     else()
         # See https://gitlab.kitware.com/cmake/cmake/-/issues/18317 for following workaround
         STRING(REGEX REPLACE "/W3" "" CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
-        ADD_COMPILE_OPTIONS(/nologo /WX /W4 /Gs32768 /D UNICODE /Gy /EHs-c- /GR- /GF /Z7 /Gw /Oy- /D_CRT_SECURE_NO_WARNINGS /wd4063 /wd4100 /wd4127 /wd4701 /wd4703 /wd4057)
+        ADD_COMPILE_OPTIONS(/nologo /WX /W4 /Gs32768 /D UNICODE /Gy /EHs-c- /GR- /GF /Z7 /Gw /Oy- /D_CRT_SECURE_NO_WARNINGS /wd4063 /wd4100 /wd4127 /wd4701 /wd4703 /wd4057 /wd4054 /wd4055)
```

Comment:
This is not related to this issue.

Please remove this change and file a new issue with new PR.